spec/system/documents/number_documents_spec.rb

```diff
@@ -19,26 +19,21 @@
     end
 
     context "when there are documents that require numbers" do
-      let!(:proposed_tag) { Document::PROPOSED_TAGS.first }
+      let!(:tag) { Document::TAGS.first }
 
       let!(:proposed_document_1) do
-        create :document, :with_file, tags: [proposed_tag],
-                                      planning_application: planning_application
+        create :document, :with_file, :with_tags,
+               planning_application: planning_application
       end
 
       let!(:proposed_document_2) do
-        create :document, :with_file, tags: [proposed_tag],
+        create :document, :with_file, tags: [Document::TAGS.first],
```

Comment:
you've updated `let!(:tag)`, but then seem to have stopped using it here? Could this not be `tags: [tag]`

Reply:
in fact: I'm not sure any of these need to have tags at all?